src/go/configgenerator/clustergen/clustergentest/clustergentest.go

```diff
@@ -12,7 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package clustergen_test
+// Package clustergentest contains test helpers to test cluster generators.
+package clustergentest
```

Comment:
Rename to `utils`?
- it helps differentiate with clustergen_test?
- It is under `clustergen/clustergentest` so it clear that it include the corresponding test helpers

Reply:
Not ideal. In golang packages imports are based on last path segment. So other libraries importing this will see it as `utils.SuccessOPTestCase`, which does not indicate cluster vs filter gen. go/go-utility-packages#good-package-names

BTW this is standard golang convention: https://stackoverflow.com/a/31794241/4402434